src/main/java/org/javarosa/core/model/FormDef.java

```diff
@@ -1343,177 +1347,16 @@ public String fillTemplateString(String template, TreeReference contextRef, Hash
         return template;
     }
 
-    /**
-     * Identify the itemset in the backend model, and create a set of SelectChoice
-     * objects at the current question reference based on the data in the model.
-     *
-     * Will modify the itemset binding to contain the relevant choices
-     *
-     * @param itemset The binding for an itemset, where the choices will be populated
-     * @param curQRef A reference to the current question's element, which will be
-     *                used to determine the values to be chosen from.
-     */
-    public void populateDynamicChoices(ItemsetBinding itemset, TreeReference curQRef) {
```

Comment:
@shubham1g5 Shout out again for refactoring this code out. Anything that drops the size of the FormDef object by 200 lines is a huge improvement to the readability of the code.